updatecli/policies/open-release-pr/updatecli.d/open-release-pr.yaml

```diff
@@ -0,0 +1,164 @@
+name: Open a PR that prepares the policy for release
+pipelineid: "{{ .pipelineid }}"
+
+sources:
+  newestGitHubDraftReleaseVersion:
+    name: Extract version from newest GitHub draft release from release drafter
+    kind: githubrelease
+    spec:
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repo }}"
+      token: "{{ requiredEnv .github.token }}"
+      key: "title"
+      typefilter:
```

Comment:
If you only specify draft: true then it will only looks for draft releases.
You may want to add prerelease and release

Reply:
We specifically only want the last draft release, created with release-drafter, as its title has the prospective semver that we want to release.

Reply:
Done it with https://github.com/kubewarden/automation/pull/99/commits/b82c6fbd7b708d4ba07030cd891de4aff7b75562.